benchmarks/README.md

```diff
@@ -0,0 +1,193 @@
+# Description
+
+This folder contains benchmark running scenarios and their resulting outputs continuously. If you want to execute them, you can do so by using the following command:
+
+```bash
+python -m benchmarks
+```
+
+The benchmarking module relies on callables that encapsulate the usage of BayBE code in general. The callables are defined in the domains folder, where each domain and every benchmark case has its own file. This file contains the complete definition and all components of a benchmark and is the place where benchmark-related objects, as well as BayBE-related code, are defined. The following components are relevant for a benchmark:
+
+## Benchmarking Structure
+
+The following describes the structure and logic of the benchmarking module, followed by an example of how to create a benchmark.
+
+### Benchmark
+
+```mermaid
```

Comment:
All of these `mermaid` blocks look really weird to me, and I also do not think that it is necessary. People can just look at the corresponding classes. If you want to keep something like that, please adjust it such that it is proper python style (applies to all of this).